### DISCUSSION TITLE
Configure the transcodes path from environment variables

### DISCUSSION THREAD

**Discussion by jooola** [jellyfin/jellyfin#9335] [Q&A]
I am looking for a way to configure the transcodes directory path using environment variables, is that currently possible ? If not how could I work around this ?

I know I can change it from the UI, but it is inconvenient, when deploying a jellyfin instance to have to setup things in the UI while it could be configured from the env just like the others paths (cache/config/logs).

**Comment by justin13888** [comment]
I wasn't able to find an answer on the doc: https://jellyfin.org/docs/general/administration/configuration#cache-directory

But since I run mine on Docker, I just mounted it directly to the default transcode directory (i.e. /config/transcodes for official docker image)
